test/build.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -l
```

Comment:
Does it work without -l?  That would be better.

Reply:
If possible I think it's best to design the script to be independent of the shell invocation selected. So it's best to eliminate any dependence on the environment settings at the time the script is called.